examples/react-app/src/FormElements/index.tsx

```diff
@@ -1,10 +1,17 @@
 import { useState } from 'react';
-import { PlmgTab, PlmgTabs, PlmgRadioButtonGroup } from '@ducky/plumage-react';
+import { PlmgTab, PlmgTabs } from '@ducky/plumage-react';
 import Sliders from './Sliders';
 import TextForm from './TextForm';
 import Radios from './Radios';
+import { SLIDERS } from './Constants/Sliders';
 
 export default function FormElements() {
+  const [sliders, setSliders] = useState({
+    decimal: SLIDERS.decimal.default,
+    stepped: SLIDERS.stepped.default,
+    large: SLIDERS.large.default,
+  });
```

Comment:
Why did you choose to move setting the slider state to the parent component?

Reply:
Because changing tabs was making the slider values reset. When the tabs changed, the SlidersForm component re-rendered causing the value to be reset. The alternative I could see was using a context provider.